### PR TITLE
Add danger labels to delete commands

### LINKED ISSUE #206: [PE-D][Tester D] Unclear UG. Would be better to seperate it out in a Warning box
-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: ryamgoh/ped#5

### PR DESCRIPTION
**Pull request by aslam341**
Closes #206.

### CODE REVIEW

**Review by timleow — APPROVED** [review]
LGTM!

**Review by Chandan8186 — CHANGES_REQUESTED** [review]
LGTM, but I think the `clear` command would also benefit from having a `Danger` label!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -425,6 +425,15 @@ Clears all entries from TaskHub.
 
 Format: `clear`
 
+<div markdown="block" class="alert alert-danger">
+
+**:no_entry: Danger**<br>
+
+* This action is irreversible.
+* Please ensure that you are certain that you want to all entries before proceeding.
```
Comment (Chandan8186) [inline docs/UserGuide.md:433]: The word `delete` to be added. Maybe it would be better to specify that all employees, projects and tasks will be deleted?

**Review by Chandan8186 — CHANGES_REQUESTED** [review]
Just a minor change requested!

File: docs/UserGuide.md (hunk 1 above)
Comment (aslam341) [reply docs/UserGuide.md]: Done

**Review by Chandan8186 — APPROVED** [review]
LGTM!

**Pull request merged**